packages/blade/src/components/Button/BaseButton/StyledBaseButton.native.tsx

```diff
@@ -8,7 +8,10 @@ import { useTheme } from '~components/BladeProvider';
 
 const StyledPressable = styled(Animated.createAnimatedComponent(Pressable))<
   Omit<StyledBaseButtonProps, 'onClick' | 'accessibilityProps'>
->((props) => getStyledBaseButtonStyles(props));
+>((props) => ({
+  ...getStyledBaseButtonStyles(props),
+  alignSelf: 'center',
```

Comment:
Need to let the native button align itself to center otherwise it takes up all the available space and spreads itself